src/screens/regimen_setup/index.tsx

```diff
@@ -23,12 +50,30 @@ export default function RegimenSetupScreen(props: RegimenSetupScreenProp) {
   const { colors } = useThemeContext();
   const { navigation } = props;
 
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  const sliderRef = useRef<{ goToSlide(index: number): void }>(null);
+
+  const handleNextButton = () => {
+    const nextScrollIndex = currentSlide + 1;
+    sliderRef.current.goToSlide(nextScrollIndex);
+    handleSlideChange(nextScrollIndex);
+  };
+
+  const handlePreviousButton = () => {
+    const previousScrollIndex = currentSlide - 1;
+    sliderRef.current.goToSlide(previousScrollIndex);
+    handleSlideChange(previousScrollIndex);
+  };
+
+  const handleSlideChange = (index: number) => setCurrentSlide(index);
+
   return (
     <SafeAreaView>
       <Header
         title={() => (
           <HeaderTitleContainer>
-            <HeaderTitle>Regimen Question 1</HeaderTitle>
+            <HeaderTitle>Regimen Question {currentSlide + 1}</HeaderTitle>
             <HeaderTitleOf>of</HeaderTitleOf>
             <HeaderTitleNumber>10</HeaderTitleNumber>
```

Comment:
the total number of slides is not up to 10. so what you will do is to use the length of the questions instead of hard coding the last slide

Reply:
removed hard-coded HeaderTitleNumber, replaced with length of slides